Themes/default/css/index.css

```diff
@@ -2783,18 +2783,21 @@ dl.addrules dt.floatleft {
 	background: rgba(40, 64, 80, 0.5);
 	z-index: 6;
 }
-.popup_window {
+.popup_window,
+#main_menu .popup_window,
+#genericmenu .popup_window {
 	position: relative;
-	width: 480px;
+	width: auto;
 	z-index: 99;
-	padding: 0 6px 6px 6px;
```

Comment:
This padding is the bit that I would like to see restored.

Reply:
ohh that makes sense